tutorial_dataset_4.qmd

```diff
@@ -121,17 +121,15 @@ References
 11 Drake et al. 2011      
 ```
 
-You would now need to look up each of these references in the reference section of the manuscript and use Google Scholar (or another reference resource) to look up the doi for each reference.\
-
-To add additional references, you need to add an argument to the `metadata_add_source_doi` function:\
+You would now need to look up each of these references in the reference section of the manuscript and use Google Scholar (or another reference resource) to look up the doi for each reference.\To add additional references, you need to add an argument to the `metadata_add_source_doi` function:\
 
 ```{r, eval=FALSE}
-metadata_add_source_doi(dataset_id = "tutorial_dataset_4", doi = "10.1071/bt9960249", type = "secondary")
+metadata_add_source_doi(dataset_id = "tutorial_dataset_4", doi = "10.1071/bt9960249", type = "original_01")
```

Comment:
Does that work? I thought the function only accepted types "primary" and "secondary".